agbenchmark/config.json

```diff
@@ -0,0 +1 @@
+{"workspace": "workspace"}
```

Comment:
one of these config.json will be necessary for testing, I stil dont know which one.
The idea is: wherever you are running agbenchmark is where the config.json should be.
And I think based on your changes @Swiftyos , we should run agbenchmark in the root of the repo.

So in the code let's use cwd as the reference for the config.json